### PR TITLE
PAPI conformance tests and eager size retrieval

### PR DESCRIPTION
**Pull request by Horneth**
- Prefix default locations in CWL with `gs://...` root for PAPI conformance tests
- Retrieve size of files early to avoid unnecessary I/O (there's still too much redundant I/O but it's a step)
- Uses `WomObject` to map back JS objects instead of `WomMap` that needs homogoneous value type (or it ends up being `WomAnyType`

### CODE REVIEW

File: wom/src/main/scala/wom/expression/NoIoFunctionSet.scala [hunk 1]
```diff
@@ -23,7 +24,7 @@ class EmptyIoFunctionSet extends IoFunctionSet {
 
   override def size(path: String): Future[Long] = Future.failed(new NotImplementedError("size is not available here"))
 
-  override implicit def ec: ExecutionContext = null
+  override implicit def ec: ExecutionContext = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(1))
```
[the remark this answers is not in the document]
Comment (Horneth) [reply wom/src/main/scala/wom/expression/NoIoFunctionSet.scala]: I was getting NPEs in the tests. I don't remember where exactly but I think it's getting accessed in a `map` even if the functions are not defined. I'll re-take a look

[the remark this answers is not in the document]
Comment (Horneth) [reply wom/src/main/scala/wom/expression/NoIoFunctionSet.scala]: I removed it from there and only added in the test instances that need it

File: wom/src/main/scala/wom/expression/NoIoFunctionSet.scala [hunk 2]
```diff
@@ -23,7 +24,7 @@ class EmptyIoFunctionSet extends IoFunctionSet {
 
   override def size(path: String): Future[Long] = Future.failed(new NotImplementedError("size is not available here"))
 
-  override implicit def ec: ExecutionContext = null
+  override implicit def ec: ExecutionContext = null//ExecutionContext.fromExecutor(Executors.newFixedThreadPool(1))
```
Comment (danbills) [inline wom/src/main/scala/wom/expression/NoIoFunctionSet.scala]: looks like an artifact

Comment (Horneth) [reply wom/src/main/scala/wom/expression/NoIoFunctionSet.scala]: oops yep

**Pull request merged**